### PR TITLE
Al-amin/28392/Changed url for about us pages

### PR DESCRIPTION
**Pull request by alamin-deriv**
Changes:

-   Leadership Page added
- Our Story Page added
- Changed Our story link -> "about/#story" -> "about/story"
- Changed Leadership link -> "about/#leadership" -> "about/leadership"

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — DISMISSED** [review]

File: src/components/custom/other-platforms.js [hunk 1]
```diff
@@ -394,13 +394,13 @@ export const NavCompany = ({ onClick }) => (
...
             onClick={onClick}
-            to="/about/#story"
+            to="/about/story/"
         />
         <CardLink
             icon={() => <img src={Leadership} alt="leadership" width="24" height="24" />}
             title={localize('Our leadership')}
             onClick={onClick}
-            to="/about/#leadership"
+            to="/about/leadership"
```
Comment (sean-binary) [inline src/components/custom/other-platforms.js:403]:
```suggestion
            to="/about/leadership/"
```

File: src/components/elements/off-canvas-menu.js [hunk 2]
```diff
@@ -366,13 +366,13 @@ export const OffCanvasMenuWrapper = (props) => {
                         header_style={header_style}
                         style={content_style}
                     >
-                        <StyledLink to="/about/#story" onClick={handleArrowClick}>
+                        <StyledLink to="/about/story" onClick={handleArrowClick}>
```
Comment (sean-binary) [inline src/components/elements/off-canvas-menu.js:369]:
```suggestion
                        <StyledLink to="/about/story/" onClick={handleArrowClick}>
```

Comment (sean-binary) [reply src/components/elements/off-canvas-menu.js:369]: And for the rest of it as well 👍

**Comment by sean-binary** [comment]
We're gonna have duplicated content if we have it this way. Instead, let's have a redirection from `/about` to `/about/story` to handle the 404 pages. That's why we will only require one story page content. Let's have a chat about this @alamin-deriv 😄

**Comment by alamin-deriv** [comment]
> We're gonna have duplicated content if we have it this way. Instead, let's have a redirection from `/about` to `/about/story` to handle the 404 pages. That way, we will only require one story page content. Let's have a chat about this @alamin-deriv 😄

File: src/components/custom/other-platforms.js (hunk 1 above)
Comment (sean-binary) [reply src/components/custom/other-platforms.js:403]: Think this was missed out.

File: src/pages/leadership/index.js [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            <SEO
+                title={localize('About Us | An Online Trading Platform | Deriv.com')}
+                description={localize(
+                    'Deriv.com - A Binary.com brand, is a pioneer and award-winning online trading platform in the trading market.',
+                )}
+            />
+            <Background>
+                <StyledContainer direction="column">
+                    <Header as="h1" type="display-title" color="white" align="center">
+                        {localize('Our Leadership')}
```
Comment (sean-binary) [inline src/pages/leadership/index.js:111]:
```suggestion
                        {localize('Our leadership')}
```

File: src/pages/story/index.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            <SEO
+                title={localize('About Us | An Online Trading Platform | Deriv.com')}
+                description={localize(
+                    'Deriv.com - A Binary.com brand, is a pioneer and award-winning online trading platform in the trading market.',
+                )}
+            />
+            <Background>
+                <StyledContainer direction="column">
+                    <Header as="h1" type="display-title" color="white" align="center">
+                        {localize('Our Story')}
```
Comment (sean-binary) [inline src/pages/story/index.js:61]:
```suggestion
                        {localize('Our story')}
```

Comment (sean-binary) [reply src/pages/story/index.js:61]: This should be small caps to match the previous string used, so that the translations are able to detect it.

**Review by sean-binary — APPROVED** [review]

**Comment by sean-binary** [comment]
Rest looks good to me. Great work 👍

**Review by sean-binary — APPROVED** [review]

**Review by yashim-deriv — APPROVED** [review]
LGTM

**Pull request merged**